src/applications/disability-benefits/all-claims/pages/addDisabilities.js

```diff
@@ -32,6 +32,14 @@ export const uiSchema = {
         {
           'ui:options': {
             freeInput: true,
+            inputTransformers: [
+              input => input.replace(/["”']/g, '’'),
+              input => input.replace(/[;]/g, '--'),
```

Comment:
What do you think of surrounding this with spaces too? We might get kinda funny-looking output otherwise.

Before
```
Separated by semi-colon; two clauses
```
After
```
Separated by semi-colon-- two clauses
```
Alternative
```
Separated by semi-colon -- two clauses
```

Reply:
Ah yeah, that seems nice